feature-libs/product-configurator/rulebased/components/overview-filter-bar/configurator-overview-filter-bar.component.spec.ts

```diff
@@ -300,5 +324,127 @@ describe('ConfiguratorOverviewFilterBarComponent', () => {
 
       expect(inputConfig).toEqual(expectedInputConfig);
     });
+
+    describe('getNextElementIdToFocusForAttributeFilter', () => {
+      it('should return the element ID when indexOfRemoved is less than the length of attrFilters', () => {
+        const attrFilters = ['filter1', 'filter2', 'filter3'];
+        const result = component['getNextElementIdToFocusForAttributeFilter'](
+          1,
+          attrFilters
+        );
+        expect(result).toBe(PREFIX_ID + attrFilters[1]);
+      });
+
+      it('should return null when removed attribute filter was the last one in the list', () => {
+        const attrFilters = ['filter1', 'filter2', 'filter3'];
+        const result = component['getNextElementIdToFocusForAttributeFilter'](
+          3,
+          attrFilters
+        );
+        expect(result).toBeNull();
+      });
+
+      it('should return null when attrFilters array is empty', () => {
+        const attrFilters: string[] = [];
+        const result = component['getNextElementIdToFocusForAttributeFilter'](
+          0,
+          attrFilters
+        );
+        expect(result).toBeNull();
+      });
+    });
+
+    describe('getNextElementIdToFocusForGroupFilter', () => {
+      it('should return the next element id when indexOfRemoved is less than the length of groupFilters', () => {
```

Comment:
```suggestion
      it('should return the next element id if indexOfRemoved is less than the length of groupFilters', () => {
```
Please change other test descriptions as well

Reply:
done